triggerEfficiencyStudy/masterJSON.json

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
I'll move this one to https://github.com/cericeci/SUEPCoffea_dask/tree/ZH_dev/data where all other corrections are